packages/vite/src/client/overlay.ts

```diff
@@ -7,9 +7,36 @@ declare const __HMR_CONFIG_NAME__: string
 const hmrConfigName = __HMR_CONFIG_NAME__
 const base = __BASE__ || '/'
 
-// set :host styles to make playwright detect the element as visible
-const template = /*html*/ `
-<style>
+// Create an element with provided attributes and optional children
+function h(
+  e: string,
+  attrs: Record<string, string> = {},
+  ...children: (string | Node)[]
+) {
+  const elem = document.createElement(e)
+  for (const [k, v] of Object.entries(attrs)) {
+    if (k === 'class') {
+      elem.className = v
+    } else {
+      elem.setAttribute(k, v)
+    }
+  }
+
+  // Append children
+  for (const child of children) {
+    // If a child is a string, create a text node
+    if (typeof child === 'string') {
+      elem.appendChild(document.createTextNode(child))
+    } else {
+      // Otherwise, append the child node
+      elem.appendChild(child)
+    }
+  }
+
```

Comment:
```suggestion
  elem.append(...child)
```
I think we can use `append` here.